src/gui/gps/qgsgpsmarker.h

```diff
@@ -24,10 +24,10 @@
 class QPainter;
 
 /**
- * \ingroup app
+ * \ingroup gui
  * A class for marking the position of a gps pointer.
  */
```

Comment:
Since it's newly added to the api, you can mark the since version as 3.12:

```suggestion
 * \since QGIS 3.12
 */
```